js_modules/dagster-ui/packages/ui-core/src/graphql/schema.graphql

```diff
@@ -2756,7 +2769,16 @@ type PartitionRun {
   run: Run
 }
 
-type PartitionBackfill {
+type PartitionBackfill implements RunsFeedEntry {
+  runId: String!
+  runStatus: RunStatus!
```

Comment:
This schema looks great but giving backfills a `runId` feels a bit icky -- I wonder if we could call the common interface's id and status just `id` and `status`, or potentially make new `entryId` and `entryStatus`?

Alternatively it might be nice to put a Grafene description on these fields to clarify that they're just meant to implement this common interface

Reply:
id and status are already taken (by the backfill id and bulk action status respectively). My hesitation with adding `entryId` and `entryStatus` is that we'd have to add those to `GrapheneRun` as well. I was hoping to keep `GrapheneRun` as unchanged as possible so that in the future when we've eliminated the need for backfills entirely, the `GrapheneRun` doesn't have attributes that are there because of an interface that is no longer necessary

Reply:
I'll definitely add descriptions though. I'm also open to pushback on not doing `entryId/Status`. If it adds significant confusion/code smell it might not be worth it

Reply:
@alangenfeld  were you thinking we'd switch GraphenRun.id of GraphenePartitionBackfill.id to either string or ID so that the types match?

Reply:
> the status one i think is useful since it flattens to just one enum

can you elaborate on this a bit?

Reply:
> were you thinking we'd switch GraphenRun.id of GraphenePartitionBackfill.id to either string or ID so that the types match?
 
I assumed they were the same, i think its fine to change backfill.id to `ID` to align them, pretty sure that a safe change since ID is String

> can you elaborate on this a bit?

as opposed to the id change above, we cant change the return type of either existing status field so we need to add a new field to return one enum type on both objects. That assumes that we want that which @bengotow can speak to